### ISSUE TITLE
Add all defaultProps to Spinner component

### ISSUE THREAD

**Issue by thatvegandev** [chakra-ui/chakra-ui#6805]
### Description

I'd like to be able to override all the defaultProps of a Spinner, but am currently only able to override size

### Link to Reproduction

https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/spinner.ts

### Steps to reproduce

The [docs](https://chakra-ui.com/docs/components/spinner/props) define numerous default props for a the spinner component, but the [theme source](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/spinner.ts) only lets you override the size prop. Is there a reason for this?

### Chakra UI Version

2.3.5

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by anubra266** [comment]
@thatvegandev If I understand your question correctly, you're referring to `colorScheme` and `variant`
Let's look at the docs again:
<img width="820" alt="CleanShot 2022-10-13 at 11 23 06@2x" src="https://user-images.githubusercontent.com/30869823/195572316-336b8b76-0658-4b6a-933c-fec3f247c9d1.png">

Note the "not implemented in the default theme" what's in the theme source is the default theme, you don't find something there doesn't mean it can't be added. That's why the other part ends with "You can extend the theme to implement them"

In short, if you want `colorSchemes` or `variants` in Spinner, you can customize the theme and add it. The guide for that is here: https://chakra-ui.com/docs/styled-system/customize-theme
